infer/models/c/src/libc_basic.c

```diff
@@ -682,6 +682,9 @@ unsigned sleep(unsigned seconds) {
 #ifdef __APPLE__ // define __WAIT_STATUS as int * on mac
 #define __WAIT_STATUS int*
 #endif
+#ifdef __GNUC__  // define __WAIT_STATUS as int * with glibc
```

Comment:
I would prefer to check for #ifndef __WAIT_STATUS instead of redefining __WAIT_STATUS (I think it gets defined on older platforms)